modules/GEPanelApp.py

```diff
@@ -0,0 +1,248 @@
+import re
+import csv
+import requests
+from ontoma import OnToma
+from settings import Config
+from collections import OrderedDict
+from common.HGNCParser import GeneParser
+from common.GCSUtils import GCSBucketManager
+
+import logging
+logger = logging.getLogger(__name__)
+
+__copyright__ = "Copyright 2014-2018, Open Targets"
```

Comment:
i will forever be against including these `__uselessinformation__` strings in the code. In my mind this information is already contained in the LICENSE file, the README and in the git commit histories. Adding it here will only mean that it will soon become out of date.

what difference does it make if the `__status__` is not production?